### DISCUSSION TITLE
Which parameter controls LeafCoverTotal in Barley/wheat model?

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9043] [Q&A]
Tillering is said to cease once a threshold fraction of incoming radiation has been achieved to captures shading effects on tillering. In the current barley/wheat model, the threshold is 0.5. That is to say, `Branching rate` will be 0 when the `LeafCoverTotal` is equal or greater than 0.5.

the `LeafCoverTotal`  reached the peak value at ZS31 and maintained this level until the mid grain-filling stage then started to decrease till maturity.
I tried to change Extinction Coefficient but the `LeafCoverTotal` value did not change too much. 

So how could we manipulate the `LeafCoverTotal`? and what factors control LeafCoverTotal?

**Comment by hut104** [comment]
CoverTotal includes cover by both green and senesced/dead material.  This uses the both the live and dead LAI values and the extinction coefficients for live and dead material.  As the crop moves past flag leaf, most covertotal will be driven by the dead leaf material.
